tern/formats/spdx/spdxtagvalue/generator.py

```diff
@@ -159,133 +160,135 @@ def get_license_block(license_list):
-    image is not applicable in the SPDX case as it is an empty image
-    object with no metadata as nothing got built.
-    The whole document should be stored in a string which can be written
-    to a file using the write_report function in report.py
-    First convert the image object into a dictionary. The dictionary
-    should be in this form:
-        image:{
-          origins: [...]
-          layers: [
-            {origins: [...],
-             packages: [
-               {origins: [...], package1: {...}},
-               {origins: [...], package2: {...}}...]}, ...]}
-    Then convert this into a flat format starting from top to bottom
-    So:
-        ## image
-        List all the tag-values here
-        make a PackageComment: <text> </text>
-
-        ## relationships
-        spdx-ref CONTAINS layer1
-        spdx-ref CONTAINS layer2
-        ...
-
-        ## layer1
-        List all the tag-values here
-        make a PackageComment here
-
-        # relationships
-        spdx-ref CONTAINS package1
-        spdx-ref CONTAINS package2
-        ....
-
-        # layer2
-        tag-values
-        PackageComment
-
-        # relationships
-        spdx-ref HAS_PREREQUISITE layer1
-        spdx-ref CONTAINS package3
-        spdx-ref CONTAINS package4
-
-        ....
-
-        # package1
-        tag-values
-        PackageComment
-
-        # package2
-
-        # package3
-
-        # package4
-
-
-    Everything in Origins can be in a tag-value format as
-    PackageComment: <text> </text>
-
-    For the sake of SPDX, an image is a 'Package' which 'CONTAINS'
-    each layer which is also a 'Package' which 'CONTAINS' the real Package'''
-    report = ''
-    licenses_found = []  # This is needed for unrecognized license strings
-    image_obj = image_obj_list[0]
-    template = SPDX()
-    # The image's PackageDownloadLocation is from a container registry
-    # This includes all the layers but the packages' download location
-    # is unknown if the download_url is blank
-    registry_repotag = image_obj.get_download_location() if hasattr(
-        image_obj, 'repotag') else 'NOASSERTION'
-
-    # first part is the document tag-value
-    # this doesn't change at all
-    report = report + get_document_block(image_obj) + '\n'
-
-    # this part is the image part and needs
-    # the image object
-    report = report + get_main_block(
-        image_obj.to_dict(template),
-        image_obj.origins.origins,
-        SPDXID=get_image_spdxref(image_obj),
-        PackageLicenseDeclared='NOASSERTION',
-        PackageLicenseConcluded='NOASSERTION',
-        PackageCopyrightText='NOASSERTION',
-        FilesAnalyzed='false') + '\n'
-    # Add image relationships
-    report = report + get_image_relationships(image_obj) + '\n'
-
-    # Add the layer part for each layer
-    for index, layer_obj in enumerate(image_obj.layers):
-        # this is the main block for the layer
+
+class Spdxtagvalue(generator.GenerateReport):
```

Comment:
Could you rename this class to `SpdxTagValue` following CamelCase?

Reply:
Done.